lib/backends/cpu/ops/upsample.ts

```diff
@@ -19,6 +19,21 @@ export class CpuUpsample extends Upsample {
   }
 }
 
+export class CpuUpsampleV9 extends UpsampleV9 {
+  run(inferenceHandler: CpuInferenceHandler, inputs: Tensor[]): Tensor[] {
+    const scales = inputs[1].floatData;
```

Comment:
since the scales are dynamic now, should we check the length?

Reply:
Should be resolved